### DISCUSSION TITLE
Example 02 pyplot error

### DISCUSSION THREAD

**Discussion by fpxno1** [NatLabRockies/floris#574] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/57779494/218738008-5fa35b92-690b-461e-a851-985531e44ad9.png)

I have installed the floris but when I run the example 02_vatialization.py ,it turns out "TypeError: vars() argument must have __dict__ attribute"

**Comment by rafmudaf** [comment]
Hi @fpxno1

Please share more details such as:
- Python version
- FLORIS version
- How you installed FLORIS
- How you obtained the examples

Unfortunately, this is not a known issue, so any other details that would be useful to explain your process to someone without context would be very helpful.

Thanks!

**Reply by fpxno1** [reply]
python version :3.8.16
FLORIS version:3.2.2
I install floris through pip install floris
I obtained the examples by downloading the source code in github
Thank you very much

**Reply by Bartdoekemeijer** [reply, marked as answer]
The error seems to be coming from `matplotlib`. Could you verify that other (i.e., non-FLORIS) python scripts with `matplotlib` do work?

**Reply by rafmudaf** [reply]
Could you also share the version of matplotlib? You can get this with `pip freeze` and find the line for matplotlib.
